app/models/appeal.rb

```
@@ -111,7 +111,7 @@ def latest_attorney_case_review
   end
 
   def reviewing_judge_name
-    task = tasks.order(created_at: :desc).detect { |t| t.is_a?(JudgeTask) }
+    task = tasks.where.not(status: "cancelled").where(type: "JudgeDecisionReviewTask").order(created_at: :desc).first
```

Let's use `Constants.TASK_STATUSES.cancelled` and `JudgeDecisionReviewTask.name` here so that the application will fail at compile time (in the second example) if we ever decide to change the strings to which those variables refer instead of silently returning no tasks.